lambdas/sys_user_management/src/app.py

```diff
@@ -32,22 +33,22 @@ def __call__(self, event):
 
         logger.info("Called the Processor")
 
-        trigger = event.get('action', [])
+        trigger = event.get('action', '')
 
         match trigger:
             case ['create']:
```

Comment:
your cases should not be lists, but strings.

Reply:
Ooops, sorry, forgot to clear brackets